values.yaml

```diff
@@ -218,6 +218,27 @@ postgres:
   # the PostgreSQL connection. For acceptable values, see:
   #  https://www.postgresql.org/docs/9.1/libpq-ssl.html
   sslMode: "require"
+  # postgres.ssl -- Options for configuring the SSL cert, key, and root cert
+  # when connecting to Postgres.
+  ssl:
+    # postgres.ssl.certSecret -- Secret containing a PEM encoded cert file.
+    certSecret:
+      # postgres.ssl.certSecret.name -- Name of the secret.
+      name: ""
+      # postgres.ssl.certSecret.key -- Key pointing to a certificate in the secret.
+      key: ""
+    # postgres.ssl.keySecret -- Secret containing a PEM encoded key file.
+    keySecret:
+      # postgres.ssl.keytSecret.name -- Name of the secret.
```

Comment:
longer-term, we might want to consider an option to load this from a file in the certs directory (e.g. load the file into a Secret and use that secret name automatically)... but this is good for now